pages/interface/components/Footer/index.js

```diff
@@ -48,7 +48,9 @@ export default function Footer(props) {
             paddingX: [2, null, null, 5],
             flexWrap: 'wrap',
           }}>
-          <Link href="https://github.com/filipedeschamps/tabnews.com.br">GitHub</Link>
+          <Link href="https://github.com/filipedeschamps/tabnews.com.br" target="_blank">
```

Comment:
Não sei se isso será implementado por causa do comentário  https://github.com/filipedeschamps/tabnews.com.br/issues/477#issuecomment-1182110074, mas caso seja implementado, acho importante ter um ícone indicando que abrirá numa nova guia, porque isso foge do comportamento padrão do sistema.

Algumas referências:

* [How to indicate this button/link opens a new tab?](https://ux.stackexchange.com/q/55419/137760)
* [Should we tell users that a link or button will open in a new tab or window & how to do so?](https://ux.stackexchange.com/q/79127/137760)

Reply:
Creio que voce tem razao, mesmo adicionando um underline ao link do Github isso iria fugir do padrao do sistema.

Reply:
Creio que voce tem razao, mesmo adicionando um underline ao link do Github isso iria fugir do padrao do sistema.

Reply:
Devo fechar este PR ou apenas remover essa parte do target="_blank", assim poderia aproveitar a parte da correcao do mobile design. @Rafatcb

Reply:
Como são duas coisas diferentes, acho que você pode deixar a correção do design mobile aqui e criar um issue sobre o link do GitHub (e, se quiser, já deixar um PR aberto com a implementação, com o risco de não ser mergeado).

Reply:
@Rafatcb Pode deixar, vou subir um commit desfazendo esses ajustes do Link